client/blocks/product-plan-overlap-notices/index.jsx

```diff
@@ -99,52 +92,52 @@ class ProductPlanOverlapNotices extends Component {
 		return availableProducts[ currentPlanSlug ].product_name;
 	}
 
-	getOverlappingFeatureName() {
-		const { availableProducts } = this.props;
-
-		if ( ! this.hasOverlap() ) {
-			return null;
-		}
-
-		const currentProductSlug = this.getCurrentProductSlug();
-		if ( ! currentProductSlug ) {
-			return null;
-		}
-
-		const productsShortNames = getJetpackProductsShortNames();
-		if ( productsShortNames[ currentProductSlug ] ) {
-			return productsShortNames[ currentProductSlug ].toLowerCase();
-		}
+	getProductItem( productSlug ) {
+		const { purchases } = this.props;
+		const productPurchase = purchases.find( purchase => purchase.productSlug === productSlug );
 
-		if ( availableProducts[ currentProductSlug ] ) {
-			return availableProducts[ currentProductSlug ].product_name;
+		if ( ! productPurchase ) {
+			return false;
 		}
 
-		return null;
+		return (
+			<li key={ productSlug }>
+				<a href={ managePurchase( productPurchase.domain, productPurchase.id ) }>
+					{ this.getProductName( productSlug ) }
+				</a>
+			</li>
+		);
 	}
 
 	render() {
 		const { selectedSiteId, translate } = this.props;
+		const overlappingProductSlugs = this.getOverlappingProducts();
 
 		return (
 			<Fragment>
 				<QuerySitePlans siteId={ selectedSiteId } />
 				<QuerySitePurchases siteId={ selectedSiteId } />
 				<QueryProductsList />
 
-				{ this.hasOverlap() && (
+				{ 0 !== overlappingProductSlugs.length && (
 					<Notice
 						showDismiss={ false }
-						status="is-warning"
 						text={ translate(
-							'Your %(planName)s Plan includes %(featureName)s. ' +
-								'Looks like you also purchased the %(productName)s product. ' +
-								'Consider removing %(productName)s.',
+							'Your %(planName)s Plan includes:' +
+								'{{list/}}' +
+								'Consider removing conflicting products.',
 							{
 								args: {
-									featureName: this.getOverlappingFeatureName(),
 									planName: this.getCurrentPlanName(),
-									productName: this.getCurrentProductName(),
+								},
+								components: {
+									list: (
+										<ul>
```

Comment:
Suggestion: We might want to apply some bottom margin to this unlabeled list for better vertical flow. 

This is how it looks with `margin-bottom: 0.5em;` as an example:

<img width="1071" alt="Screen Shot 2020-03-31 at 6 08 02 PM" src="https://user-images.githubusercontent.com/4044428/78086246-974f9180-737a-11ea-9e5d-eadd99095d56.png">

Reply:
Yeah, noticed that too. Let me add that.

Reply:
Resolved with 506826a913fe5830e3c1122b02453add71e09466.